build.gradle

```diff
@@ -50,19 +47,26 @@ shadowJar {
   zip64 true
   configurations = [project.configurations.shadow]
   from(project.sourceSets.main.output)
-  archiveName = "brotli-codec-${version}.${extension}"
+  archiveFileName = "brotli-codec-${archiveVersion}.${archiveExtension}"
 }
 
-install.dependsOn shadowJar
+publish.dependsOn shadowJar
 
 dependencies {
-  shadow 'org.meteogroup.jbrotli:jbrotli:0.5.0'
-  shadow 'org.meteogroup.jbrotli:jbrotli-native-linux-x86-amd64:0.5.0'
-  shadow 'org.meteogroup.jbrotli:jbrotli-native-darwin-x86-amd64:0.5.0'
+  shadow 'com.aayushatharva.brotli4j:brotli4j:1.3.2-SNAPSHOT'
```

Comment:
Temporarily uses a SNAPSHOT until https://github.com/hyperxpro/Brotli4j/pull/15 is merged.